### PR TITLE
feat(sessions): extend sorting and filtering

### PR DESCRIPTION
**Pull request by RichardKruemmel**

### CODE REVIEW

File: web/src/components/table/use-cases/sessions.tsx [hunk 1]
```diff
@@ -176,8 +176,10 @@ export default function SessionsTable({
     {
       accessorKey: "userIds",
       enableColumnFilter: !omittedFilter.find((f) => f === "userIds"),
+      id: "userIds",
```
Comment (maxdeichmann) [inline web/src/components/table/use-cases/sessions.tsx:179]: image
I just saw that in our seed:load command we end up in a state where we have a sessionid with traces null. Can you check what is going on to me? It should be at least 1 in my understanding. Can be another ticket as this is not really related to this change.

File: web/src/components/table/use-cases/sessions.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ export default function SessionsTable({
   const userIdFilter: FilterState = userId
```
Comment (maxdeichmann) [inline web/src/components/table/use-cases/sessions.tsx:58]: image

Do you know why the selection in the dropdown is not selected?

File: web/src/server/api/definitions/sessionsView.ts [hunk 3]
```diff
@@ -3,38 +3,75 @@ import { type ColumnDefinition } from "@/src/server/api/interfaces/tableDefiniti
...
     id: "id",
     type: "string",
     internal: 's."id"',
   },
   {
-    name: "userId",
+    name: "User Ids",
+    id: "userIds",
     type: "string",
     internal: "array_to_string(t.\"userIds\", ', ')",
```
Comment (maxdeichmann) [inline web/src/server/api/definitions/sessionsView.ts:15]: What do you think about re-using the tags filter here? I think having a drop down with userids would be most helpful.

File: web/src/components/table/use-cases/sessions.tsx (hunk 2 above)
Comment (RichardKruemmel) [reply web/src/components/table/use-cases/sessions.tsx:58]: Found the issue and fixed it. 👍

Comment (RichardKruemmel) [reply web/src/components/table/use-cases/sessions.tsx:58]: Fixed both

File: web/src/components/table/use-cases/sessions.tsx (hunk 1 above)
Comment (RichardKruemmel) [reply web/src/components/table/use-cases/sessions.tsx:179]: After short investigation I believe that this session didn't get created by the seeder as the syntax of the seeder script is: `session-${i % 10}`.

File: web/src/components/table/use-cases/sessions.tsx [hunk 4]
```diff
@@ -176,8 +189,10 @@ export default function SessionsTable({
     {
       accessorKey: "userIds",
       enableColumnFilter: !omittedFilter.find((f) => f === "userIds"),
-      header: "User ID",
+      id: "userIds",
+      header: "User IDs",
       enableHiding: true,
+      enableSorting: true,
```
Comment (maxdeichmann) [inline web/src/components/table/use-cases/sessions.tsx:195]: How does sorting for multiple user Ids work?

File: web/src/components/table/use-cases/sessions.tsx [hunk 5]
```diff
@@ -282,6 +302,7 @@ export default function SessionsTable({
       id: "usage",
       header: "Usage",
       enableHiding: true,
+      enableSorting: true,
```
Comment (maxdeichmann) [inline web/src/components/table/use-cases/sessions.tsx:305]: Same as traces table. Does this work?

Comment (RichardKruemmel) [reply web/src/components/table/use-cases/sessions.tsx:305]: Yes it does. Usage column behaves internally like total_token column. As reference the sessionTable.ts defintion:
`  {
    name: "Usage",
    id: "usage",
    type: "number",
    internal: 'o."totalTokens"',
  },`
